kratos/tests/cpp_tests/utilities/test_element_size_calculator.cpp

```diff
@@ -138,6 +172,42 @@ namespace Kratos
                 ElementSizeCalculator<2, 4>::AverageElementSizeDerivative, 1e-8, 1e-7);
         }
 
+        KRATOS_TEST_CASE_IN_SUITE(Quadrilateral2D9MinimumElementSizeDerivatives, KratosCoreFastSuite)
+        {
+            Geometry<NodeType>::PointsArrayType nodes;
+            nodes.push_back(NodeType::Pointer(new NodeType(1, 0.0, 0.0, 0.0)));

```

Comment:
Avoid push_back, define the vector with the final size and assign later. Push_back is inefficient. The same for the others

Reply:
I just followed the format of the script. The nodes vector is created using this way in all tests.